docs/staging-use-cases.md

```diff
@@ -191,7 +188,7 @@ Multitenancy Management
 
 - As a multi-cluster operator, I would like to provide an EKS-like experience in which the workload control plane nodes are joined to the management cluster and the control plane config isn’t exposed to the consumer of the workload cluster. This enables me as an operator to manage the control plane nodes for all clusters using tooling like prometheus and fluentd. I can also control the configuration of the workload control plane in accordance with business policy.
 
-### Multitenancy Management
+### Multitenancy for Management Clusters
```

Comment:
since this is already under the "Operator of Management Cluster" section, isn't adding "for management clusters" redundant?  Also, reading the cases, I feel like it's not really multitenancy for the management cluster, but more multitenancy for workload clusters but from the point of view of a management cluster operator

Reply:
My reading is that this section is about about having multiple users with rights on the management cluster e.g. cluster creation, deletion etc. The above section (L153) is about preventing workload cluster users from interfering with the management cluster.

The main reason for changing this was to stop the clash between this multitenancy management and the duplicate on L153 which previously broke the link.

 Any name change will do - let me know what you think will work well here.

Reply:
On testing it looks like the multitenancy-management-1 link works on github, but it's not part of every markdown so it wasn't working for me locally. I'll revert this